### PR TITLE
Reorganize directory structure

### PR DESCRIPTION
**Pull request by oliverdunk**
Reorganizes the overall directory structure and updates the README accordingly. We've discussed these changes internally and feel they will tidy up the top-level of the repository and make it easier to find things.

Associated PR is coming for developer.chrome.com to update any references.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,18 +5,16 @@ Note that Chrome Apps are deprecated. Learn more [on the Chromium blog](https://
...
-* [examples/](examples/) - full featured extensions spanning multiple API packages
-* [apps/](apps/) - deprecated Chrome Apps platform (not listed below)
-* [mv2-archive/](mv2-archive/) - resources for manifest version 2
+* [api-samples/](api-samples/) - extensions focused on a single API package
+* [functional-samples/](functional-samples/) - full featured extensions spanning multiple API packages
+* [_archive/apps/](_archive/apps/) - deprecated Chrome Apps platform (not listed below)
+* [_archive/mv2/](_archive/mv2/) - resources for manifest version 2
 
 To experiment with these samples, please clone this repo and use 'Load Unpacked Extension'.
 Read more on [Getting Started](https://developer.chrome.com/extensions/getstarted).
```
Comment (AmySteam) [inline README.md:20]:
```suggestion
Read more on [Development Basics](https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked).
```
I wanted to send developers directly to the "Load unpacked" instructions.

**Review by AmySteam — APPROVED** [review]
@oliverdunk Thanks for this! 👍 Its a huge improvement.

Just one quick request on the readme and it's ready to go

**Review by Danyalkasiri — APPROVED** [review]
****

**Review by Mannyflow7 — APPROVED** [review]

**Comment by Sarrac3873** [comment]
> Reorganizes the overall directory structure and updates the README accordingly. We've discussed these changes internally and feel they will tidy up the top-level of the repository and make it easier to find things.
> 
> Associated PR is coming for developer.chrome.com to update any references.

**Comment by Sarrac3873** [comment]
> Reorganizes the overall directory structure and updates the README accordingly. We've discussed these changes internally and feel they will tidy up the top-level of the repository and make it easier to find things.
> 
> Associated PR is coming for developer.chrome.com to update any references.

**Review by sb2409 — APPROVED** [review]

**Comment by rdvnnl** [comment]
inalridvan@gmail.com

**Pull request merged**